config/services.php

```diff
@@ -36,5 +36,8 @@
 
     'twitter' => [
         'widget_id' => env('TWITTER_WIDGET_ID'),
+    ],
+    'sparkpost' => [
+        'secret' => env('SPARKPOST_SECRET')
```

Comment:
Then it should be in an independent commit

On Sun, 29 May 2016 13:22 JapSeyz, <notifications@github.com> wrote:

> In config/services.php
> <https://github.com/Attendize/Attendize/pull/115#discussion_r65009260>:
>
> > @@ -36,5 +36,8 @@
> >
> >      'twitter' => [
> >          'widget_id' => env('TWITTER_WIDGET_ID'),
> > +    ],
> > +    'sparkpost' => [
> > +        'secret' => env('SPARKPOST_SECRET')
>
> That's true, it is a general fix for using SparkPost. The TransportManager
> expects that to be there.
>
> It's a bug that it is missing IMO, and thus is fixed here.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Attendize/Attendize/pull/115/files/f4b081650e76ef6452fa531ab23780bb87e06629#r65009260>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABx8OQBSkoRr3Yp-qr-NfoqcV9tsUiO-ks5qGdlXgaJpZM4IpRqU>
> .
>